src/components/add-button/index.tsx

```diff
@@ -8,11 +8,9 @@ interface AddButtonProps {
 
 export const AddButton: React.FC<AddButtonProps> = ({ onClick }) => {
   return (
-    <div className="btn-container">
-      <div className="btn-add" onClick={onClick}>
-        <div className="btn-text">Adicionar Usuário</div>
-        <Plus className="icon" size={24} weight="bold" />
-      </div>
-    </div>
+    <S.AddButton onClick={onClick}>
```

Comment:
não vejo muito ganho em importar como uma única variável, poderia importar cada componente individualmente
```suggestion
    <AddButton onClick={onClick}>
```

Reply:
Opa!

Beleza, corrigi em todos os arquivos 👍🏻 
Valeu Rafael